internal/db/schema/migrations/postgres/2/84_oidc.up.sql

```diff
@@ -363,6 +363,34 @@ create trigger
 before insert on auth_oidc_account
   for each row execute procedure insert_auth_oidc_account_subtype();
 
```

Comment:
I just added a lenghty comment on the changes to repository_account.go basically questioning the premise of enforcing this workflow on the operators.